ls1intum/Artemis
# [Feature] Display student code in new component and fix position of lock repo label

### Checklist


### Motivation and Context


We introduce a new component which displays the student code of programming exercises without using ace editor. This component is currently not in use but will then be used for the manual assessment.

### Description

- Introduce new component to display student code
- Fixed position of lock repo label

### Steps for Testing


**NOTE: You can test this only locally with the programming setup at the moment**
To test it you need to adapt the following first:
- Replace `<jhi-code-editor-ace>` component in `code-editor-container.html` with following:
`<jhi-code-editor-tutor-assessment-display-code editorCenter [selectedFile]="selectedFile" (onError)="onError($event)"></jhi-code-editor-tutor-assessment-display-code>`
- Import the new component to `code-editor.module`or you can press `alt + enter`this should do it for you 😊 


1. Log in to Artemis
2. Navigate to Course Administration
3. Open up the code editor
4. Make sure that the new component is loaded and displays the file data correctly

Syntax highlighting will be added in the future.

### Screenshots


![image](https://user-images.githubusercontent.com/41108487/91341334-a297d380-e7d9-11ea-8e45-738489c4d751.png)

![video1](https://user-images.githubusercontent.com/41108487/91341884-82b4df80-e7da-11ea-86e1-986ccb28a4a9.gif)

Position of lock repo label now:
<img width="1214" alt="image" src="https://user-images.githubusercontent.com/41108487/91694868-8eebc480-eb6d-11ea-9418-bac58926124a.png">
